edxval/admin.py

```diff
@@ -36,13 +36,40 @@ class VideoAdmin(admin.ModelAdmin):  # pylint: disable=C0111
     inlines = [CourseVideoInline, EncodedVideoInline]
 
 
+class VideoImageForm(forms.ModelForm):
+
+    course_video = forms.CharField(required=True)
+
+    class Meta:
+        model = VideoImage
+        fields = ('course_video', 'image', 'generated_images')
+
+
 class VideoImageAdmin(admin.ModelAdmin):
+    list_display = ('get_course_id', 'image', 'generated_images')
+
+    def get_course_id(self, obj):
+        return obj.course_video.course_id
```

Comment:
Shouldn't there be both course and video IDs?

Reply:
something like `course_video`

Reply:
We need to show the course id why course_video ?